helper/staking/staking.go

```diff
@@ -138,10 +147,20 @@ func PredeployStakingSC(
 		return nil, fmt.Errorf("unable to generate DefaultStatkedBalance, %w", err)
 	}
 
+	if minValidatorCount > maxValidatorCount {
+		return nil, fmt.Errorf("minimum number of validator can not be greater than maximum number of validator")
```

Comment:
This should be defined as a separate error with `errors.New`, and not with `fmt.Errorf`

Reply:
Fixed in https://github.com/0xPolygon/polygon-edge/pull/397/commits/0cd22450d81525e6724c7ecf8f4761dcd64fe0d6

Reply:
Yes, I was thinking of making this error available at the beginning of the file with other errors, like

```go
var (
    errInvalidValidatorRange = errors.new("minimum ...")
)
```

Reply:
Got it, fixed in https://github.com/0xPolygon/polygon-edge/pull/397/commits/1962fcb18003c084a232b432f728b8e14cfde03e